config_parser.cc

```diff
@@ -152,6 +152,7 @@ bool NginxConfigParser::Parse(std::istream* config_file, NginxConfig* config) {
   config_stack.push(config);
   TokenType last_token_type = TOKEN_TYPE_START;
   TokenType token_type;
+  int block_depth = 0;
```

Comment:
Maybe add a brief comment explaining what block_depth measures, and when it changes.